code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -91,6 +90,12 @@
 		if(3)
 			adjustBruteLoss(30)
 
+/mob/living/simple_animal/hostile/guardian/gib()
+	if(summoner)
+		src.summoner << "<span class='danger'><B>Your [src.name] was blown up!</span></B>"
```

Comment:
I mostly do it for myself, I just prefer to have that on there for whatever reason. I started removing them but realized how many there were and how late it was.